packages/emmett-expressjs/src/testing/apiSpecification.ts

```diff
@@ -123,16 +113,16 @@ export const ApiSpecification = {
                 const response = await handle();
 
                 if (typeof verify === 'function') {
-                  const succeded = verify(response);
+                  const succeeded = verify(response);
 
-                  if (succeded === false) assert.fail();
+                  if (succeeded === false) assert.fail();
                 } else if (Array.isArray(verify)) {
                   const [first, ...rest] = verify;
 
                   if (typeof first === 'function') {
-                    const succeded = first(response);
+                    const succeeded = first(response);
```

Comment:
@thiagomini, thanks for fixing my typos! 😅

Reply:
Sure, no problem haha I only got them because I have the "Code Spell Cheker" extension in my VSCode